modules/users/server/config/strategies/local.js

```diff
@@ -10,12 +10,16 @@ var passport = require('passport'),
 module.exports = function () {
   // Use local strategy
   passport.use(new LocalStrategy({
-    usernameField: 'username',
+    usernameField: 'usernameOrEmail',
     passwordField: 'password'
   },
-  function (username, password, done) {
+  function (usernameOrEmail, password, done) {
     User.findOne({
-      username: username.toLowerCase()
+      $or: [{
+        username: usernameOrEmail.toLowerCase()
+      }, {
+        email: usernameOrEmail
```

Comment:
Emails are always saved lowercase in the db, so it's safe to set this `sernameOrEmail.toLowerCase()` as well.